test/test_autograd.py

```diff
@@ -2020,6 +2020,36 @@ def func(x):
         gradcheck(func, [x])
         gradgradcheck(func, [x])
 
+    def test_select_scatter(self):
```

Comment:
Why not just an opinfo?

Reply:
it seems the answer is, because the other view functions don't have op infos either

Reply:
They do. I did 2 search and found view and narrow. So I guess others are there too.

Reply:
We should add these as OpInfos so that someone else doesn't need to come along and do that in the future

Reply:
Time to brush up on OpInfo authoring :)